modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less

```diff
@@ -47,12 +48,11 @@
         align-items: center;
         display: flex;
         margin-bottom: @pad-xs;
-      }
-    }
 
-    > *:last-child:not(:only-child) {
-      border-color: @border-color;
-      border-style: solid;
+        .tox-icon svg {
```

Comment:
I probably didn't expect this much of a change, but I also haven't looked into it. What's the reason for this change?

Reply:
This is to set the icons to be middle aligned. The `svg` defaults to `inline`, it has an extra space which makes the icon a bit off.

Current: 
![image](https://user-images.githubusercontent.com/111734091/195474899-1dda9438-15cb-489d-a2b5-bca572a26577.png)

Reply:
I wonder if we should split that into a separate ticket. This is a different way from how it is addressed in the notification banners, but maybe that doesn't matter. There, they seem to use grid and align-self etc. I guess it all depends on how many other `tox-icon`s we have around with this problem.